backend/kernelCI_app/typeModels/commonOpenApiParameters.py

```diff
@@ -0,0 +1,57 @@
+from drf_spectacular.utils import OpenApiParameter
+
+COMMIT_HASH_PATH_PARAM_REQUIRED = OpenApiParameter(
+    name="commit_hash",
+    description="Commit hash of the tree",
+    required=True,
+    type=str,
+    location=OpenApiParameter.PATH,
+)
+
+TREE_NAME_PATH_PARAM_REQUIRED = OpenApiParameter(
+    name="tree_name",
+    description="Name of the tree",
+    required=True,
+    type=str,
+    location=OpenApiParameter.PATH,
+)
+
+GIT_BRANCH_PATH_PARAM_REQUIRED = OpenApiParameter(
+    name="git_branch",
+    description="Branch name of the tree",
+    required=True,
+    type=str,
+    location=OpenApiParameter.PATH,
+)
+
+BUILD_ID_PATH_PARAM_REQUIRED = OpenApiParameter(
+    name="build_id",
+    description="ID of the build to retrieve issues",
```

Comment:
What do you think of something like this?

```suggestion
    description="ID of the build to retrieve the issues from",
```